Samples/kotlin/src/main/res/values/integers.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <!-- TODO: Change this buildingId to match the building on MaaS -->
-    <integer name="buildingId">68616</integer>
-    <!--<integer name="buildingId">87788</integer>-->
+    <integer name="buildingId"></integer>
```

Comment:
Let's the leave the building id as default. Sample usually has 12345 as default id.

Reply:
@avakkachen Updated with default buildingId 12345